server/base.py

```diff
@@ -145,6 +145,37 @@ def removeThumbnails(event):
     ImageItem().removeThumbnailFiles(event.info)
 
 
+def prepareCopyItem(event):
+    """
+    When copying an item, adjust the largeImage fileId reference so it can be
+    matched to the to-be-copied file.
+    """
+    srcItem, newItem = event.info
+    if 'largeImage' in newItem:
+        li = newItem['largeImage']
+        for pos, file in enumerate(Item().childFiles(item=srcItem)):
+            for key in ('fileId', 'originalId'):
+                if li[key] == file['_id']:
```

Comment:
When trying this on an svs file I have, it raised `KeyError: KeyError('originalId',)`.  That key is only present for images that were retiled correct?

Reply:
Yes.  I'll change this to `if li.get(key) == file['_id']:`.